api-server/src/events/existing-event-detection-controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Body,
+  Controller,
+  HttpCode,
+  Inject,
+  LoggerService,
+  Post,
+} from '@nestjs/common';
+import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
+import ExistingEventDetectionService from './existing-event-detection-service';
+import { VERSION_1_URI } from '../utils/versionts';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import ExistingEventDetectionParameters from './dto/existing-event-detection-parameters';
+import ExistingEventDetectionResults from './dto/existing-event-detection-results';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config();
```

Comment:
Why does this need to be called here?

Reply:
I need to investigate that. To be honest I don't remember and I'm not sure if it does, but it's in all the other controllers. I seem to recall there was a weird issue with making that gets initialized. It wasn't really integrated into the nest flow that well.

I think it's fairly likely this is not needed, but I was afraid to leave it out.

I created this issue to look into it https://github.com/infinite-industries/infinite/issues/476